maven-scm-plugin/src/main/java/org/apache/maven/scm/plugin/AbstractScmMojo.java

```diff
@@ -57,6 +58,9 @@
  * @author <a href="evenisse@apache.org">Emmanuel Venisse</a>
  * @author Olivier Lamy
  */
+ 
+@Mojo( name = "ScmMojo",
```

Comment:
This adds new goal, including documentation (site, `:help`)
![scmmojo](https://user-images.githubusercontent.com/11896137/142248223-1571f28f-2b59-4d6f-b5dc-0de94bcc937b.png)
but such goal cannot be executed:
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-scm-plugin:1.11.3-SNAPSHOT:ScmMojo (default-cli) on project xavier: Execution default-cli of goal org.apache.maven.plugins:maven-scm-plugin:1.11.3-SNAPSHOT:ScmMojo failed: Unable to load the mojo 'ScmMojo' (or one of its required components) from the plugin 'org.apache.maven.plugins:maven-scm-plugin:1.11.3-SNAPSHOT': com.google.inject.ProvisionException: Unable to provision, see the following errors:
[ERROR] 
[ERROR] 1) No implementation for org.apache.maven.scm.plugin.AbstractScmMojo was bound.
[ERROR]   while locating org.apache.maven.scm.plugin.AbstractScmMojo
```